packages/inferno-router/__tests__/BrowserRouter.spec.tsx

```diff
@@ -26,12 +27,13 @@ describe('BrowserRouter (jsx)', () => {
 
     spyOn(console, 'error');
 
+    // @ts-ignore
     render(<BrowserRouter history={history} />, node);
 
     expect(console.error).toHaveBeenCalledTimes(1);
 
     // browser only?
-    expect(console.error.calls.argsFor(0)[0]).toContain('<BrowserRouter> ignores the history prop');
+    expect((console.error as any).calls.argsFor(0)[0]).toContain('<BrowserRouter> ignores the history prop');
```

Comment:
putting the result of spyOn(console, 'error'); to variable corrects the typing so "any" is not needed

Reply:
Fixed in 54581c76